packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -91,11 +91,11 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
                                 ></td>
                             </tr>
                             <tr
-                                data-ng-if="$ctrl.process.selectedOffer.modemRental"
+                                data-ng-if="['yes','recycled'].includes($ctrl.process.selectedOffer.modem)"
```

Comment:
**nitpick:** `['yes','recycled']` is duplicated in multiple places. maybe it can be moved to one common place and re-use.